src/biogeochem/CNFireLi2014Mod.F90

```diff
@@ -1266,5 +1473,23 @@ subroutine CNFireFluxes (this, bounds, num_soilc, filter_soilc, num_soilp, filte
    end associate 
 
  end subroutine CNFireFluxes
+!     subroutine vegc_fitransfer(p,ito,ifrom,default_transfer,matrix_transfer,vegcpool,fi_rate)
```

Comment:
Please clean the code and remove all the commented out lines here and other places in the code.